pkg/cosign/verify.go

```diff
@@ -867,6 +889,16 @@ func CheckExpiry(cert *x509.Certificate, it time.Time) error {
 	return nil
 }
 
+func getBundleIntegratedTime(sig oci.Signature) (time.Time, error) {
+	bundle, err := sig.Bundle()
+	if err != nil {
+		return time.Now(), err
+	} else if bundle == nil {
+		return time.Now(), nil
+	}
+	return time.Unix(bundle.Payload.IntegratedTime, 0), nil
```

Comment:
This time will be extracted from the RFC3161 timestamp for the TSA option.

Reply:
That's for rekor bundle integrated time extraction, but yes, we would need another option for extracting the validityTime from the TSA in the outer call.

Reply:
yep. It makes sense.